boilerplate/app/i18n/i18n.ts

```diff
@@ -16,7 +16,7 @@ import jp from "./jp"
 // to use regional locales use { "en-US": enUS } etc
 const fallbackLocale = "en-US"
 export const i18n = new I18n(
-  { ar, en, "en-US": en, ko, fr, jp },
+  { ar, en, "en-US": en, ko, fr, ja: jp },
```

Comment:
for consistency, please change the different files that are labeled `jp` to `ja`